pkg/manager/setconfig.go

```diff
@@ -99,29 +99,29 @@ func (m *Manager) computeNewNetworkConfig(targetUpdates map[string]devicechanget
 		//FIXME this is a sequential job, not parallelized
 		version := deviceInfo[devicetype.ID(target)].Version
 		deviceType := deviceInfo[devicetype.ID(target)].Type
-		newChange, err := m.ComputeNewDeviceChange(
```

Comment:
I thought `newChange` here was actually a good name for the **new** change

Reply:
brought back